tests/test_trajectory.py

```diff
@@ -719,16 +719,19 @@ def test_smooth(get_fn):
 
     eq(output, test)
 
-@pytest.mark.skip(reason="Broken, maybe only on Python 3.11")
 def test_image_molecules(get_fn):
     # Load trajectory with periodic box
     t = md.load(get_fn('alanine-dipeptide-explicit.dcd'), top=get_fn('alanine-dipeptide-explicit.pdb'))
     # Image to new trajectory
     t_new = t.image_molecules(inplace=False)
+    # Test for issue #1769
+    assert t_new.xyz is not t.xyz
```

Comment:
Great to see this additions! 

One issue that often occurs when imaging has been dealing with multimeric systems (ie protein+ligand) and trying to image on the entire multimer. Would you be willing to add a test of some kind that checks to ensure these changes would work with multimeric systems? Even just like 2 alanine dipeptides (or alanine dipeptide + anything) would be sufficient I imagine

Reply:
Having said that I will address this, I now realise I'm not sure I understand your comment. The alanine dipeptide example system is itself multimeric in that it includes explicit water molecules; this is explicitly tested in lines 735 onwards:
    # Image with specified anchor molecules
    molecules = t.topology.find_molecules()
    anchor_molecules = molecules[0:3]
    t.image_molecules(inplace=True, anchor_molecules=anchor_molecules)

I'm guessing you have something else in mind - can you elaborate please?

Reply:
Oh yes my bad - I should have been more precise in my description. The issue encountered by others previously has been where non-solvent atoms (like a protein+ligand or a protein+protein) system does not appropriately get  fixed (only one chain is correctly imaged, usually).

I was imagining something like a test using two alanine-dipeptides (or similarly small system) where we ensure they are both imaged appropriately, initially. 

That all said, since we already have a "multimeric system" with explicit water molecules, we should probably just test that the water molecules were appropriately corrected. 

Could you check that the anchor molecules being selected (and fixed) are the alanine-dipeptide plus two separate molecules in line 735? If so then there's no need for an additional test!

Reply:
Sure. Running interactively in the ./tests folder:
```
import mdtraj as mdt
t = mdt.load('data/alanine-dipeptide-explicit.dcd', top='data/alanine-dipeptide-explicit.pdb')
molecules = t.topology.find_molecules()
anchor_molecules = molecules[0:3]
print(anchor_molecules[0])
{ACE1-H1, ACE1-CH3, ACE1-H2, ACE1-H3, ACE1-C, ACE1-O, ALA2-N, ALA2-H, ALA2-CA, ALA2-HA, ALA2-CB, ALA2-HB1, ALA2-HB2, ALA2-HB3, ALA2-C, ALA2-O, NME3-N, NME3-H, NME3-C, NME3-H1, NME3-H2, NME3-H3}
print(anchor_molecules[1])
{HOH4-H2, HOH4-O, HOH4-H1}
print(anchor_molecules[2])
{HOH5-O, HOH5-H1, HOH5-H2}
```
So indeed the three molecules in `anchor_molecules` are the alanine dipeptide, plus two waters.

Reply:
Great! Thank you for following up on this - this means that the tests are already checking for appropriate multimer imaging!